src/overlays/actors/ovl_En_An/z_en_an.h

```diff
@@ -1,14 +1,20 @@
 #ifndef Z_EN_AN_H
 #define Z_EN_AN_H
 
+#include "unk.h"
+#include "z64actor.h"
+#include "z64msgevent.h"
+
 #include "global.h"
 #include "assets/objects/object_an1/object_an1.h"
 
 struct EnAn;
+struct Path;
+struct PlayState;
 
-typedef void (*EnAnActionFunc)(struct EnAn*, PlayState*);
-typedef void (*EnAnDialogueFunc)(struct EnAn*, PlayState*);
-typedef s32 (*MsgEventFunc)(Actor*, PlayState*);
+typedef void (*EnAnActionFunc)(struct EnAn*, struct PlayState*);
+typedef void (*EnAnDialogueFunc)(struct EnAn*, struct PlayState*);
+typedef s32 (*MsgEventFunc)(Actor*, struct PlayState*);
```

Comment:
Just a note `MsgEventFunc` shouldn't exist, there is already a typedef in `z64msgevent.h`. I plan on fixing this in a clean up PR.